frb_codegen/src/ir/ty_dart_opaque.rs

```diff
@@ -0,0 +1,40 @@
+use crate::ir::*;
+
+#[derive(Debug, Clone, Hash, Eq, PartialEq)]
+pub struct IrTypeDartOpaque;
+
+impl IrTypeTrait for IrTypeDartOpaque {
+    fn visit_children_types<F: FnMut(&IrType) -> bool>(&self, _f: &mut F, _ir_file: &IrFile) {
+        // do nothing.
+    }
+
+    fn safe_ident(&self) -> String {
+        "DartObject".to_owned()
```

Comment:
uber-nit: Sometimes we say DartOpaque, sometimes we say DartObject. Shall we unify them?